data/styles/clear/include/priorities_4_overlays.prio.txt

```diff
@@ -1166,7 +1171,10 @@ leisure-recreation_ground                           # caption z17-
 leisure-slipway                                     # caption z17-
 === 650
 
+historic-anchor                                     # icon z18- (also has caption(optional) z18-)
+historic-cannon                                     # icon z18- (also has caption(optional) z18-)
 historic-memorial-plaque                            # icon z18- (also has caption(optional) z18-)
```

Comment:
memorial plaques are so low and so low visibility because there are places with almost each house having a plaque or two...
I bet cannons and stones are much more rare, so we can use higher visibility and priority?

Reply:
They may be more rare, but I think they are roughly the same historic importance as a plaque, so I don't think it's wise to increase priority. 

I put z18- visibility because they are usually about the size of a bench, so shouldn't appear too prominently on the map (they appear at the same time as benches) but if you really think it needs raising, I can :)

Reply:
in fact, i need to lower them further (below shops) to fix the entrance issue :)